### DISCUSSION TITLE
人如何獲得幸福——我讀《心流》

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1227] [General]
《心流》是一本研究最優體驗的正向心理學書籍，作者觀察了在各領域，包含藝術、科學、運動界的傑出人士，想探尋讓這些高度卓越並具高度創造力的人們，不被名利綁架，願意持續燃燒熱情的原因。在全書的第一章節，作者不急於介紹進入心流的方法，而是利用各種統計數據與文獻，向讀者證明為什麼我們需要心流。本書以亞里斯多德的話:「無論男女，對幸福的追求都是一樣的」開篇，一句話點出自古以來人們對幸福的強烈渴望，而如何收穫並得到幸福，心流便是作者提出的答案。

這也不禁讓我開始反思，生活在物質漸豐、資源容易取得的現代，我們真的有更幸福嗎?舉個例子，過去的祖父母輩，可能時常煩惱自己的下一餐，甚至連衣食不缺的貴族，也沒有一隻能隨時解悶、收發訊息的手機，然而，現在的我們有了更多的食物、更多的娛樂，但我們有比之前的人們更加幸福、更加快樂嗎?

從以上的案例，我們可以推論出造成人們感到幸福的根本原因，從來不是外在物質的豐饒與否，任何超出基本生理需求的物質給予，不只不會讓人幸福，還造成更多的資源浪費，各式廣告時常打者幸福的幌子進行銷售，如車子的廣告影像總會搭配溫馨的家庭旅遊、快樂的蜜月旅行等等，讓人們誤以為買了這個產品後就能得到像廣告裡描述的幸福人生。

既然不能從外在獲得幸福，作者便鼓勵我們從內在去得到幸福，並強調幸福不是追求而來，而是忘卻自我的副產物，這點我深有體會，在上個學期我加入了學校的咖啡社，由於技巧不熟練，在沖煮的過程，需同時注意手的轉動、注水大小與時間控制等參數，我需要將全部的專注力放在自身以外的事物，如同和手中的沖煮壺融為一體，縱使只有短短兩三分鐘，又縱使咖啡的成品不盡理想，亦不影響令我感到幸福的沖煮過程。換句話說，令我感到幸福的主因，不是時間的長短、更不是成功與否，而是在過程中，專注而致的忘我。

總上，在我閱讀完《心流》一書後，比起各種進入身體、思想及工作心流的方式，令我最印象深刻的，仍屬重新定義幸福的第一章節，我原先認為《心流》就如同市面上許多的商管書籍一般，是一本關於自我實現、工作方法論的工具書，讀完後也確實是，卻也不只是，裡頭其實只花了短短三章進行方法的介紹，剩下的篇幅除了為讀者定義幸福外，亦鼓勵人們對抗混沌、為自己的行為創造意義等等。

一路看下來，心流像是作者窮其一生得出的，關於如何獲得幸福的暫時解答，然而，人們依舊在追尋幸福的道路上不斷前行，剩下的答案仍待我們持續探索。

**Comment by DuxtinTai** [comment]
我認為心流就像是《被討厭的勇氣2》中提到的生命時間一樣，是一種純粹、專注、不受外界干擾的享受。即使你做得不好，但只要你全心全意地投入，就會感到非常滿足。儘管心流有很多好處，但現今的生活充斥著太多的干擾，對於手機不離身的人而言，難得的心流時間也可能因為訊息導致的手機震動而被中斷。我並不是想表達手機不離身就是太過沉迷3c，而是現今社會中資訊流動的速度太快，有可能因為一不注意而失去機會。因此，我認為當今社會中，人們首先要克服的就是在資訊量繁多的情況下，保持內心的平靜，踏實地做好每一件事，不斷地進步。
